.github/workflows/integration-tests.yml

```diff
@@ -69,7 +69,7 @@ jobs:
         run: pip3 install .
```

Comment:
Can we add `.github/workflows/integration-test.yml` to the `paths` field at the beginning of this test

Reply:
like this? see my last commit.

Reply:
should be "integration-tests.yml"